graphene_django/static/graphene_django/graphiql.js

```diff
@@ -61,17 +61,29 @@
   var fetchURL = locationQuery(otherParams);
 
   // Defines a GraphQL fetcher using the fetch API.
-  function httpClient(graphQLParams) {
-    var headers = {
-      Accept: "application/json",
-      "Content-Type": "application/json",
-    };
+  function httpClient(graphQLParams, opts = { headers: {} }) {
+    let headers = opts.headers;
```

Comment:
It's rather safe to assume that this file won't be transpiled into es5-compatible version during the build process, so I'd suggest not to use default parameters (shipped in es6). Same for `let`, let's use `var` as it was before.

```suggestion
  function httpClient(graphQLParams, opts) {
    if (typeof opts === 'undefined') {
      opts = {};
    }
    var headers = opts.headers || {};
```